content/_partners/determining-your-assurance-level._en.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: partners/determining-your-assurance-level
+permalink: /partners/determining-your-assurance-level/
+title: >-
+    Determining your assurance level
+---
+
+## About assurance levels
+
+Assurance levels (also called “Service levels,” or “<a href="https://csrc.nist.gov/glossary/term/identity_assurance_level" class="external-link">Levels of Assurance</a>”) is a general term referring to the trustworthiness of a given transaction. Assurance levels are considered essential components of identity systems, due to the underlying complexity of identity verification processes. Generally, the higher the Service Levels, the greater the trustworthiness of the authentication and verification processes that occurred for a specific transaction and identity. 
```

Comment:
It'd be nice if we could mark up these links with Markdown syntax. We do have some [prior art for that](https://github.com/GSA-TTS/identity-site/blob/93956124510ac76ae86d8c80b765731a67b611c0/content/_partners/faq._en.md#L36), though it uses Kramdown's custom syntax for adding attributes to links. More ideal would be to have some way to automatically add the class to links to external domains, maybe by overriding the Markdown parsing similar to what we're doing with [adding `usa-link` class to links automatically](https://github.com/GSA-TTS/identity-site/blob/93956124510ac76ae86d8c80b765731a67b611c0/_plugins/content_typography.rb#L12).

I think it could also be a good idea to work toward using the [design system standard `usa-link--external`](https://designsystem.digital.gov/components/link/). I'm not sure why we created a duplicated ad-hoc style in the initial development of the partner site.

Reply:
Thanks!

The external links can be cleaned up. I've thought of enforcing the automatic addition of the external link class before and I'll look into that.

We can switch to the class `usa-link--external` over `external-link`. There's a slight rendering difference between the two where `external-link` is smaller.